instagram/src/components/PostsPage/PostsPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Posts from '../Posts/Posts';
+import Header from '../Header/Header';
+// import '../../App.css';
+
+const PostsPage = props => {
+    return (
+        <div className="container">
+            <Header 
+            handleInputChange={props.handleInputChange}
+            filter={props.filter}
+            logOut={props.logOut}
+            />
+            <Posts 
+            data={props.data}
+            commentText={props.commentText}
+            handleInputChange={props.handleInputChange}
+            addNewComment={props.addNewComment}
+            addLike={props.addLike}
+            removeComment={props.removeComment}
+            handleInputChangeComment={props.handleInputChangeComment}
```

Comment:
I wonder if it would be possible to reduce this props passing if you'd directly altered the data array on your App.js state rather than your solution involving new arrays.